aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
> An AIM BridgeDomain can't be reconstructed by simply retrieving it's parent object.
That was NEVER the intention of the DN property. It only serves to provide a pointer to the mapped ACI object, thus enabling the consumer to explore any additional information they are interested in by interacting directly with APIC.

> "hidden detail" for all the RS objects
Why? One of the reasons we have AIM is to hide these very ugly details from the AIM consumer. I would like to be conservative and avoid unnecessary details through the AIM APIs.

> converter to retrieve the DN  .. which limits APICAPI presence in the least number of modules (all in the aci_universe package).
Actually, I think it should be the other way around - the converter should get the DN from AIM resource instead of creating one of its own. DNs for child objects should be derived from that DN. Frankly, I'm not too concerned with apicapi presence in too many modules; in any case my intention is to tease the MO definitions out of apicapi in to AIM. So this dependency will eventually go away.